java-symbol-solver-core/src/test/resources/javaparser_expected_output/com_github_javaparser_ast_comments_CommentsParser.txt

```diff
@@ -12,7 +12,7 @@
   Line 51) CommentsCollection comments = new CommentsCollection() ==> com.github.javaparser.ast.comments.CommentsCollection
   Line 51) new CommentsCollection() ==> com.github.javaparser.ast.comments.CommentsCollection
   Line 52) int r ==> int
-  Line 54) Deque prevTwoChars = new LinkedList<Character>(Arrays.asList('z', 'z')) ==> java.util.Deque
+  Line 54) Deque prevTwoChars = new LinkedList<Character>(Arrays.asList('z', 'z')) ==> java.util.Deque<java.lang.Object>
```

Comment:
Are these two equivalent? I think they are not because you can do this:

```
Box rawBox = new Box();           // rawBox is a raw type of Box<T>
Box<Integer> intBox = rawBox;     // warning: unchecked conversion
```

But you cannot do this:

```
Box<Object> rawBox = new Box<Object>();
Box<Integer> intBox = rawBox;
```